### ISSUE TITLE
Synch from Kodi but not from Trakt

### ISSUE THREAD

**Issue by bourgois** [razzeee/script.trakt#228]
Hi

Great script that I used for quite some time with xbmc :)
But I did a fresh install using latest Kodi and script and it doesn't synch back from Trakt :(

Logs:
23:33:56 T:4720267264   DEBUG: [script.trakt] sync: [Movies Sync] Getting movie collection from Trakt.tv
23:33:56 T:4720267264   DEBUG: [script.trakt] sync: [Movies Sync] Error getting Trakt.tv movie list, aborting movie Sync.
23:34:18 T:4720267264   DEBUG: [script.trakt] sync: [Episodes Sync] Invalid Trakt.tv show list, possible error getting data from Trakt, aborting Trakt.tv collection update.
23:34:18 T:4720267264   DEBUG: [script.trakt] sync: [Episodes Sync] Error getting Trakt.tv collected show list, aborting tv show sync.

Not sure if it is a bug or my install.

Thanks for helping!

**Comment by razzeee** [comment]
Might be an api error, so it might fix itself. Unfortunately I am off for the weekend, so I  can't investigate

**Comment by nfrankish** [comment]
I too, have this error. What info would you like to assist with debugging?

**Comment by razzeee** [comment]
https://github.com/trakt/script.trakt#creating-logfiles

**Comment by nfrankish** [comment]
Well. This is interesting.
So I rebooted the openelec box to get a clean set of logs. After a reboot, it told me my Trak.TV wasnt authorized and prompted me to do it. After I attempted to authorize, it said it was, but then as reported elsewhere, in the user section it wasn't showing my userid, so i did it again, and then it showed everything fine.. 

Then it also synced perfectly. 

I do note that sometime in the last day or two it auto upgraded from 3.0.1 to 3.0.2. So maybe the bug i hit originally was only present in 3.0.1.
